noxfiles/run_infrastructure.py

```diff
@@ -235,7 +235,7 @@ def _run_tests(
     _run_cmd_or_err(
         f'echo "running pytest for conditions: {pytest_path} with environment variables: {environment_variables}"'
     )
-    coverage_arg = "--cov-report=lcov:coverage/ops-integration.lcov"
+    coverage_arg = "--cov-report=xml"
```

Comment:
good catch! I knew that one would get forgotten at some point lol

Reply:
I would have sworn I did a global search for this initially and still somehow missed this one!